config/spec-bundle.js

```diff
@@ -43,7 +43,7 @@ testing.TestBed.initTestEnvironment(
  * any file that ends with spec.ts and get its path. By passing in true
  * we say do this recursively
  */
-var testContext = require.context('../tests', true, /\.spec\.ts/);
+var testContext = require.context('../src', true, /\.spec\.ts/);
```

Comment:
Hmm not sure about this ? Why did you need to move the tests to the source folder ?